src/sagemaker/tensorflow/deploying_tensorflow_serving.rst

```diff
@@ -0,0 +1,447 @@
+you are likely to have ``./multi/model1/export/Servo/...`` and ``./multi/model2/export/Servo/...``. In both cases,
+"Servo" is the base name for the SaveModel files. When serving multiple models, each model needs a unique
+basename, so one or both of these will need to be changed. The ``/export/`` part of the path isn't needed
+either, so you can simplify the layout at the same time:
+
+.. code:: bash
+
+  mv multi/model1/export/Servo/* multi/model1/
+  mv multi/model2/export/Servo/* multi/model2/
+  rm -fr multi/model1/export
+  rm -fr multi/model2/export
+
+You should now have a directory structure like this:
+
+::
+
+  └── multi
+    ├── model1
+    │   └── <version number>
+    │       ├── saved_model.pb
+    │       └── variables
+    │           └── ...
+    └── model2
+        └── <version number>
+            ├── saved_model.pb
+            └── variables
+                └── ...
+
+To repackage the files into a new archive, use ``tar`` again:
+
+.. code:: bash
+
+  tar -C "$PWD/multi/" -czvf multi.tar.gz multi/
+
+The ``multi.tar.gz`` file is now ready to use.
+
+**Uploading the new archive to S3**
+
+.. code:: bash
+
+  aws s3 cp multi.tar.gz s3://mybucket/models/multi.tar.gz
+
+Creating and Deploying a SageMaker Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the remaining steps, let's return to python code using the SageMaker Python SDK.
+
+.. code:: python
+
+  from sagemaker.tensorflow.serving import Model, Predictor
+
+  # change this to the name or ARN of your SageMaker execution role
+  role = 'SageMakerRole'
+
+  model_data = 's3://mybucket/models/multi.tar.gz'
+
+  # For multi-model endpoints, you should set the default model name in
+  # an environment variable. If it isn't set, the endpoint will work,
+  # but the model it will select as default is unpredictable.
+  env = {
+    'SAGEMAKER_TFS_DEFAULT_MODEL_NAME': 'model1'
+  }
+
+  model = Model(model_data=model_data, role=role, framework_version='1.11', env=env)
+  predictor = model.deploy(initial_instance_count=1, instance_type='ml.c5.xlarge')
+
+The ``predictor`` object returned by the deploy function is ready to use to make predictions
+using the default model (``model1`` in this example).
+
+Creating Predictor instances for different models
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``predictor`` returned by the ``model.deploy(...)`` function can only send requests to
+the default model. To use other models deployed to the same Endpoint, you need to create
+additional ``Predictor`` instances. Here's how:
+
+.. code:: python
+
+  # ... continuing from the previous example
+
+  # get the endpoint name from the default predictor
+  endpoint = predictor.endpoint
+
+  # get a predictor for 'model2'
+  model2_predictor = Predictor(endpoint, model_name='model2')
+
+  # result is prediction from 'model2'
+  result = model2_predictor.predict(...)
+
+Making predictions with the AWS CLI
```

Comment:
If I am not mistaken this doesn't work with localmode. I think we should make that clear here.

Reply:
good catch. there is a way to make it work with local mode, but it requires a change to local mode. I will add that.

Reply:
fixed